docs/00_introduction/package-pipeline.md

```diff
@@ -0,0 +1,44 @@
+# The Garden Linux Package Pipeline
+
+Garden Linux is based on [Debian](https://www.debian.org), but it has a unique design for how packages are built.
+This pipeline might look overly complex on first sight, but is based on it's requirements and past experience.
+This document aims to explain the decisions and tradeoffs made.
+
+## The bigger picture
+
+Garden Linux release artifacts are disk images for various platforms such as kvm, cloud providers or bare metal hardware.
+To build those artifacts, Garden Linux makes use of an [APT repository](https://wiki.debian.org/DebianRepository).
+APT is the Debian package management tool.
+
+Unlike Debian, Garden Linux is focussed on containers.
+This means that for users of Garden Linux, the APT repo has much less direct importance.
+Typically, we don't expect our users to install packages from the APT repository.
```

Comment:
Not relevant. The package pipeline design also works in case we would advertise the apt repos to be used for live-upgrades. which technically should work already if you consider a few configurations (immutability and apt sources).